engines/dfc_provider/spec/support/example_json.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+module ExampleJson
+  def self.read(name)
+    pathname = DfcProvider::Engine.root.join("spec/support/#{name}.json")
```

Comment:
Am I right to say that example files are normally called "fixtures"? In the main app, we actually have two fixtures folders:
1. `spec/fixtures/files/`
2. `spec/support/fixtures/`

Hmm, but also "fixtures" seems to be a more specific concept in ActiveRecord, for storing sample data in YAML in the `test/fixtures`. 

🤷 I think at least some dedicated directory would be a good idea to keep `spec/support/` more organised. My preference would be `#`1, what do you think?

Reply:
I was wondering that, too. These files are not ActiveRecord "fixtures". `spec/fixtures/files` makes that distinction and I would be okay with that. Currently, we have only three or four files in `spec/support` of that engine though. It's very tidy. When it becomes more, we can move it. And then we need to only change the ExampleJson module.

Reply:
My preference would be 1. `spec/fixtures/files/` as well